cocos/2d/CocosStudioExtension.cpp

```diff
@@ -0,0 +1,13 @@
+#include "CocosStudioExtension.h"
```

Comment:
why this file is created under the `2d` folder?  CocosStudio has its own folder.